src/shared-components/avatar/__snapshots__/test.tsx.snap

```diff
@@ -12,7 +12,6 @@ exports[`<Avatar /> UI snapshot render with size large styles 1`] = `
 <img
   alt="avatar"
   className="emotion-0 emotion-1"
-  size="large"
```

Comment:
`Avatar` was one of the few components that has a somewhat-breaking change: previously we just used a `styled.img` export directly, but this PR turns it into a react component that receives props and then passes it onto a `styled.img` component. The difference is that the Story can receive type enforcement. I also renamed the prop we pass to the `styled.img` component `avatarSize` so that we wouldn't also pass it to the DOM--this pattern (see `inline` in SVG components) is one of our biggest offenders when it comes to browser console warnings/errors.